<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SEAcret</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="encryptBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionRectangle"/>
    <addaction name="actionLasso"/>
    <addaction name="actionEraser"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionMedian_Blur"/>
    <addaction name="actionInpaint"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionChange_Median_Blur_Kernel"/>
    <addaction name="actionChange_Eraser_Size"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTool"/>
   <addaction name="menuStyle"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionLasso">
   <property name="text">
    <string>Lasso</string>
   </property>
  </action>
  <action name="actionMedian_Blur">
   <property name="text">
    <string>Median Blur</string>
   </property>
  </action>
  <action name="actionInpaint">
   <property name="text">
    <string>Inpaint</string>
   </property>
  </action>
  <action name="actionChange_Median_Blur_Kernel">
   <property name="text">
    <string>Change Median Blur Kernel Size</string>
   </property>
  </action>
  <action name="actionChange_Eraser_Size">
   <property name="text">
    <string>Change Eraser Size</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
